### DISCUSSION TITLE
umi4路由支持页面缓存吗

### DISCUSSION THREAD

**Discussion by yaodong5063** [umijs/umi#8373] [Ideas]
umi4路由支持页面缓存吗

**Comment by fz6m** [comment]
虽然目前 react 没有完美的页面缓存解决方案，但是你可以看看这个是否满足你的需求：

https://github.com/umijs/umi-presets-pro/blob/589c48244bd453644f3429361b996bd756e9decc/packages/umi-preset-pro/src/index.ts#L11

或者等待 react 18 off screen

**Reply by yaodong5063** [reply]
react18有什么新的api支持页面级缓存吗
